### PR TITLE
Dockerised front end and added Github action to store container in AW…

### PR DESCRIPTION
**Pull request by craigd2387**
…S

### CODE REVIEW

File: .vscode/launch.json
Comment (eoinmcbennett) [inline .vscode/launch.json:1]: Is this file to do with one of your extensions?

File: .vscode/tasks.json
Comment (eoinmcbennett) [inline .vscode/tasks.json:1]: Is this file to do with one of your extensions?

**Review by eoinmcbennett — CHANGES_REQUESTED** [review]

File: .DS_Store
Comment (shaunganley) [inline .DS_Store:1]: get this out of here

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
```
Comment (shaunganley) [inline .github/workflows/deploy.yml]: remember to remove these lines before merging as we only want to deploy to aws when everything is approved

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+    - name: Login to Amazon ECR
+      id: login-ecr
+      uses: aws-actions/amazon-ecr-login@v1
+
+    - name: Build & push the Docker image
+      env:
+        API_URL: ${{ secrets.API_URL }}
+        ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
+        ECR_REPOSITORY: moves-like-swagger-ui
```
Comment (shaunganley) [inline .github/workflows/deploy.yml]: this should be the same as example from last week

**Review by shaunganley — APPROVED** [review]

**Review by shaunganley — CHANGES_REQUESTED** [review]
change repository name

**Review by eoinmcbennett — APPROVED** [review]

**Review by shaunganley — APPROVED** [review]

**Pull request merged**